public/r2/v1/swagger/swagger.json

```diff
@@ -14,18 +14,14 @@
     "tags": [
         {
             "name": "namespaces",
-            "description": "Operations at the namespaces level"
+            "description": "Operations at the namespace level"
```

Comment:
per our offline chat, look into golang swagger annotation based generator

Reply:
Definitely worth doing. Will make an issue on this project.

Reply:
https://github.com/m3db/m3ctl/issues/11